Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```

Comment:
> Don't use metal rough spheres. Is there a more real-world model that can replace it and still show off the lighting well?

Why not? The idea is to show how the image-based lighting changes when the metallic/roughness parameters change. I think it might be better to have a model and change those parameters, but I'm not sure.

Reply:
I think we just try to stray away from test models in general. For PBR we switched from the the sci-fi helmet to the ground vehicle for this reason. I just realized we don't have any boat models. Maybe there is a good one out there that has a good mix of metallic and roughness.

Reply:
I was browsing Sketchfab and this helicopter model seemed pretty good. It has smooth metallic areas which is nice. We would probably want to shrink some of the textures and apply Draco compression though.

https://sketchfab.com/models/dca5ae065c584817bf59c53a0920f785.

<img width="685" alt="heli" src="https://user-images.githubusercontent.com/915398/48801944-ada40280-ecdb-11e8-86a8-2525b22d0b5b.PNG">

Reply:
A couple more suggestions for the demo

* Do you still plan on adding IBL parameters to the Entity API? That would help simplify the code in the demo.
* The "Use procedural image" checkbox is checked when it's using the environment map. It should be the reverse.
* Disable the slider if not using the procedural sky. The 3D Models Coloring demo has an example of that with knockout.
* Like in the other comment, it might be nice to add controls for `imageBasedLightingFactor` and `lightColor` to get the full sweep of lighting effects.
* Now that I think about it, would it make sense to combine this demo into the Physically-Based materials demo? It could have a drop down using the two models. We don't have to do it, but just a thought.